packages/block-library/src/navigation-menu/index.php

```diff
@@ -117,11 +117,10 @@ function build_navigation_menu_html( $block, $colors ) {
 /**
  * Register the navigation menu block.
  *
- * @uses render_block_navigation_menu()
+ * @uses gutenberg_render_block_navigation()
  * @throws WP_Error An WP_Error exception parsing the block definition.
  */
-function register_block_core_navigation_menu() {
-
+function register_block_core_navigation() {
```

Comment:
Can anyone tell me why this can't be prefixed with `gutenberg_`?

Reply:
They are prefixed, but not in the source. The prefix is added at build time:

https://github.com/WordPress/gutenberg/blob/6fc0b70/webpack.config.js#L129-L139

You can see this in the build output:

https://plugins.trac.wordpress.org/browser/gutenberg/trunk/build/block-library/blocks/navigation-menu.php

It's done this way because core copies these files directly from Gutenberg:

https://github.com/WordPress/wordpress-develop/blob/4ebaf2b/tools/webpack/packages.js#L96-L108

Since Gutenberg needs to replace / augment the implementation of core blocks, and in order to avoid redeclaring the function, a prefix is added automatically.

As far as what I was suggesting with my comments in the previous pull request, we should write the source as though we expect these functions to exist one day in core, and as such consider whether `build_css_colors` is named specific enough given that it's used only by this one block (and not a generalized utility for building CSS colors).

Reply:
I did not know that Gutenberg would do that. In that case the prefix should probably change to something like `block_{blockname}_*`